src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/SymUnmanagedReaderExtensions.cs

```diff
@@ -21,56 +21,59 @@ internal static class SymUnmanagedReaderExtensions
             var importStringGroups = reader.GetCSharpGroupedImportStrings(methodToken, methodVersion, out externAliasStrings);
             Debug.Assert(importStringGroups.IsDefault == externAliasStrings.IsDefault);
 
-            if (importStringGroups.IsDefault)
+            ArrayBuilder<ImmutableArray<ImportRecord>> importRecordGroupBuilder = null;
```

Comment:
I figured I'd avoid the extra local variable (since we'd still need a local variable for the builder).  The builder could be scoped to the if in that case, but still an extra local.